edx_solutions_api_integration/courses/tests.py

```diff
@@ -2537,6 +2546,14 @@ def test_courses_users_list_valid_email_enroll_user(self):
         response = self.do_post(test_uri, {'email': self.users[0].email})
         self.assertEqual(response.status_code, 201)
 
+    @mock.patch("edx_solutions_api_integration.courses.views.get_course", _fake_get_course)
```

Comment:
Can we simplify this to something
```
@mock.patch("edx_solutions_api_integration.courses.views.get_course", (Mock(), Mock(), None))
```
to avoid writing a method (`_fake_get_course`)